src/ui/manage-ui/src/FamilyHubs.ServiceDirectory.Admin.Web/Pages/manage-services/Service-Detail.cshtml

```diff
@@ -256,7 +256,7 @@
         @if (Model.Flow == JourneyFlow.Edit && Model.FamilyHubsUser.Role is RoleTypes.LaManager or RoleTypes.LaDualRole)
         {
             <h2 class="govuk-heading-m">Delete this service</h2>
-            <p class="govuk-!-margin-bottom-9">You can <a href="/manage-services/delete-service">delete this service</a> if you want to permanently remove it from the directory.</p>
+            <p class="govuk-!-margin-bottom-9">You can <a href="/manage-services/delete-service?serviceId=@Model.ServiceModel.Id">delete this service</a> if you want to permanently remove it from the directory.</p>
```

Comment:
Since Stu isn't in today I resolved in: [760176c](https://github.com/DFE-Digital/fh-services/pull/126/commits/760176cb686d12d1cac3391a4d74bbb6128a42b8)

Reply:
Formatting was weird so just tabbed it back to the right place: [2b57e74](https://github.com/DFE-Digital/fh-services/pull/126/commits/2b57e7454955164f704f104b9a0169f357fd484a)